airflow/hooks/presto_hook.py

```diff
@@ -20,11 +20,13 @@ class PrestoHook(BaseHook):
     >>> ph.get_records(sql)
```

Comment:
No worries!
I'll defer to @mistercrunch on this one. Because of the integration with a lot of different systems/dbs, it is usually hard to run all of them on the same machine and have them all succeed. It usually requires running each test subset on a machine that has the proper systems installed, or they error out... I wish we had a better system, but it is difficult to mock/fake every single interface.

Amazing PR though.